runway/cfngin/hooks/aws_lambda.py

```diff
@@ -455,12 +456,17 @@ def dockerized_pip(
 
     LOGGER.info('using docker image "%s" to build deployment package...', docker_image)
 
+    docker_run_args = {}
+    if _kwargs.get("python_dontwritebytecode"):
```

Comment:
since kwargs is being used now, the proceeding underscore should be removed as it indicates an unused variable.

Reply:
Legit embarrassing that I copied and pasted and didn't even think about that.